main_test.go

```diff
@@ -16,3 +16,29 @@ func TestShouldFail(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestShouldFailNavInAnotherModule(t *testing.T) {
+	errors := check("fixture", "AnotherModule", "/modules/AnotherModule/nav.adoc")
+
+	if len(errors) == 1 {
+		error := errors[0]
+		if error != "one.adoc" {
+			t.Fail()
+		}
+	} else {
+		t.Fail()
+	}
+}
+
+func TestShouldFailDocument(t *testing.T) {
```

Comment:
This has potential for an awful lot of code duplication. In go it is common to use table driven tests. See https://ieftimov.com/posts/testing-in-go-table-driven-tests/. This allows you to merge this test with the one above into a single test function.

Reply:
Good suggestion, but I'll leave it as it is for now. Will use this approach for future tests.